python/tvm/meta_schedule/utils.py

```diff
@@ -114,7 +113,10 @@ def __init__(self, *args, **kwargs):
 
         def __getattr__(self, name: str):
             """Bridge the attribute function."""
-            return self._inst.__getattribute__(name)
+            try:
+                return self._inst.__getattribute__(name)
+            except AttributeError:
+                return super(TVMDerivedObject, self).__getattr__(name)
```

Comment:
Not really related to the change here. But is it possible to make mypy treat DerivedObject as sub class of the real base class from `_tvm_metadata`? For example, right now `RPCRunner` is not a subclass of `Runner` from mypy's perspective.

Reply:
I don't have much idea how to fix the false positive warnings from mypy. Any suggestions?

Reply:
I think it's a [known bug](https://stackoverflow.com/questions/69330379/how-to-properly-type-hint-a-class-decorator) of mypy, which haven't been fixed. I didn't see better solution than mark as ignore though.

Reply:
Yeah I was troubled by this issue too, but didn't end up finding a good solution. Given it's a known issue in mypy, probably the best thing we could do is to wait and see if it's going to be fixed in next releases.